[DOCS] CLI Clean-up (#7904)
great-expectations/great_expectations: docs/docusaurus/docs/guides/validation/validation_actions/how_to_update_data_docs_as_a_validation_action.md

Test that your new Action is configured correctly:

> Run the Checkpoint from your code to verify that no errors are thrown.

Reviewer: ```suggestion
Run the following command to run the Checkpoint and verify that no errors are returned:
```
